src/cli/main.c

```diff
@@ -44,10 +44,7 @@ static bool print_usage(int argc, char **argv)
         const char *id = NULL;
         const SubCommand *command = NULL;
 
-        if (argc > 1) {
```

Comment:
Isn't this hit for clr-boot-manager help help help ... type cases?

Reply:
Huuum, didn't consider that case. I'll change the patch in a way that we simplify the code and still comply with this use case.